app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -1,438 +1,482 @@
 @import 'variables';
 @import 'mixins';
 
+#onboarding-container {
+  min-height: calc(100vh - 350px);
```

Comment:
Where is `350px` coming from?

Reply:
It was already here before I started working on this.

Reply:
We will likely end up creating our own modal components with Crayons, etc. but I didn't want to change too much just yet!

Reply:
Oh, gotcha, I thought it was an addition :) All good!